src/main/java/org/pkcs11/jacknji11/UBigInt.java

```diff
@@ -0,0 +1,24 @@
+package org.pkcs11.jacknji11;
```

Comment:
Could you add a copyright notice like the other files?  You are welcome to include yourself as the copyright owner, but license the code using the same MIT.  You are also welcome to assign copyright to me if you want to use the exact same lines from other files (I dont' really care), or you could assign copyright to a vague term like "jacknji11 authors".

Reply:
done